woocommerce/woocommerce
# Add category by ID instead of name when creating a product in e2e tests

### Changes proposed in this Pull Request:

This PR updates creating a simple product with category to use the category ID instead of the category name.

Closes https://github.com/woocommerce/woocommerce/issues/30868.

### How to test the changes in this Pull Request:

1. Verify CI passes
2. Create a simple test that creates a product with a category that wasn't previously created:

```javascript
const { createSimpleProductWithCategory } = require( '@woocommerce/e2e-utils' );

const runCreateProductWithCategoryTest = () => {
	describe('Create a product with a category', () => {
		it('should be able to create a product with a category', async () => {
			await createSimpleProductWithCategory('My test product' + '10.00', 'e2e');
		});
	});
};

module.exports = runCreateProductWithCategoryTest;
```

3. Run the test and verify the product is created with the correct category.
4. Repeat the test with the category created initially and verify the same category is applied with no errors.
